Formula/ship.rb

```diff
@@ -0,0 +1,28 @@
+class Ship < Formula
+  desc "Reducing the Overhead of Maintaining 3rd-party Applications in Kubernetes"
+  homepage "https://ship.replicated.com"
+  url "https://github.com/replicatedhq/ship/archive/v0.29.0.tar.gz"
+  sha256 "ff4598a4f2811af9894036a3e2e433c458b75ce48633012ed3b76315e15c8de1"
+
+  depends_on "go" => :build
+  depends_on "node" => :build
+  depends_on "yarn" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    srcpath = buildpath/"src/github.com/replicatedhq/ship"
+    srcpath.install buildpath.children
+    srcpath.cd do
+      system "make", "VERSION=#{version}", "build-minimal"
+      bin.install "bin/ship"
+    end
+  end
+
+  test do
```

Comment:
as mentioned before, this is very primitive checks, can we do `ship init` and check that some files been created/modified?

From the [docs](https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula):

> We want tests that don’t require any user input and test the basic functionality of the application. For example `foo build-foo input.foo` is a good test and (despite their widespread use) `foo --version` and `foo --help` are bad tests. However, a bad test is better than no test at all.

Reply:
Thanks, @igas -- I've added a simple test of the base 'ship init' functionality, following the suggestions from the docs (docs referenced examples https://github.com/Homebrew/homebrew-core/blob/master/Formula/tinyxml2.rb and https://github.com/Homebrew/homebrew-core/blob/master/Formula/cmake.rb)